components/gitpod-db/src/typeorm/entity/db-installation-admin.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { InstallationAdmin } from "@gitpod/gitpod-protocol";
+import { Entity, Column, PrimaryColumn } from "typeorm";
+import { TypeORM } from "../typeorm";
+
+@Entity()
+export class DBInstallationAdmin implements InstallationAdmin {
+  @PrimaryColumn(TypeORM.UUID_COLUMN_TYPE)
+  id: string;
+
+  @Column()
+  sendTelemetry: boolean;
```

Comment:
As mentioned by @JanKoehnlein yesterday we should thrive for as little "potential change surface" as possible in `gitpod-db` and `server` to avoid the cross-team overhead as much as possible.

I suggest a colum like this:
```
@Column("simple-json")
settings: InstallationAdminSettings;
```